### DISCUSSION TITLE
컴퓨터 네트워크(진-> 재민)

### DISCUSSION THREAD

**Discussion by jjinleee** [tempdku/temp#9] [General]

**Comment by dku19jam** [comment]
1. 어떤 상황에서 TCP를 사용해야 하고, 어떤 상황에서 UDP를 사용해야하나요
- 애플리케이션 개발 시 빠르고 많은 양의 데이터 처리 요청이 들어오는 서비스를 개발 할 경우엔 UDP를 사용해야 하며(온라인 게임, 멀티 미디어 재생 같이 반응이 중요한 경우에 사용) 데이터가 손실되지 않는 것이 보장되어야 하는 경우에는 TCP를 사용해야합니다(아마 나머지 전부 다 사용해야 할듯?)
2. 웹 브라우저가 쿠키를 사용하여 사용자를 식별하는 방법은 무엇인가요
- 브라우저에 저장 되어있는 사용자에 대한 정보의 키-밸류 쌍을 읽어와 서버에서 사용자에 대한 인증 정보가 맞는지 확인하는 방식입니다.(책에 없었던거 같아서 아는것 까지 대답해봤어용)
